temba/locations/migrations/0004_auto_20151120_0848.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import mptt.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0003_strip_boundary_alias_name'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='adminboundary',
+            name='lft',
+            field=models.PositiveIntegerField(
+                default=0, editable=False, db_index=True),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='adminboundary',
+            name='rght',
```

Comment:
same reason as the `lft` above, it is added internally by Django Mptt and used for building the hierarchy